src/assets/data/programs.json

```diff
@@ -43,5 +43,23 @@
         "description": "SINCE 2018 we are organizing this development-oriented conference web, with the idea that lecturers from Spanish speaking can expose about new technologies and transmit their community experience. Starting November 18.",
         "url": "https://webconf.tech/",
         "end_date": "2022-11-18"
+    },
+    {
+        "title": "Web Summit",
+        "description": "Politico said we run “the world’s premier tech conference”, the Atlantic that Web Summit is “where the future goes to be born”, and Reuters that we assemble “the biggest names in tech and business”.",
+        "url": "https://websummit.com/rio/",
+        "end_date": "2023-05-01"
+    },
+    {
+        "title": "SÃO PAULO TECH WEEK",
+        "description": "Através de um evento totalmente dinâmico, a SPTW será 100% online e trará uma abordagem totalmente inovadora, onde cada membro poderá gerar conexões qualificadas para os seus negócios não apenas durante a semana de palestras, mas também ao longo do ano. ",
```

Comment:
Amazing! But I will need you to put the descriptions in English.

Reply:
All rigth, thanks for your suggest. I will solve it!